.github/workflows/buildcheck.yml

```diff
@@ -31,8 +31,9 @@ jobs:
       with:
         repository: willuhn/jameica
         path: jameica
+        ref: 'JAMEICA_2_10_BRANCH'
```

Comment:
Die Branches sind ziemlich alt. Ich würde die release tags verwenden, so wie in https://github.com/openjverein/jverein/issues/516#issuecomment-2518616013 beschrieben.